Procurement/Controls/FragmentStash.xaml

```diff
@@ -0,0 +1,87 @@
+                           DataContext="{Binding Dusk}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="168" Canvas.Top="151"
+                           DataContext="{Binding Dawn}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="104" Canvas.Top="151"
+                           DataContext="{Binding Noon}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="168" Canvas.Top="91"
+                           DataContext="{Binding Midnight}" />
+
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="379" Canvas.Top="86"
+                           DataContext="{Binding Grief}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="441" Canvas.Top="86"
+                           DataContext="{Binding Rage}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="379" Canvas.Top="151"
+                           DataContext="{Binding Ignorance}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="441" Canvas.Top="151"
+                           DataContext="{Binding Hope}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="275" Canvas.Top="91"
+                           DataContext="{Binding Divine}" Height="94" />
+
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="104" Canvas.Top="400"
+                           DataContext="{Binding Volkuur}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="168" Canvas.Top="400"
+                           DataContext="{Binding Eber}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="104" Canvas.Top="463"
+                           DataContext="{Binding Inya}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="168" Canvas.Top="463"
+                           DataContext="{Binding Yriel}" />
+
+
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="379" Canvas.Top="400"
+                           DataContext="{Binding Hydra}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="441" Canvas.Top="400"
+                           DataContext="{Binding Phoenix}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="379" Canvas.Top="463"
+                           DataContext="{Binding Minotaur}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="441" Canvas.Top="463"
+                           DataContext="{Binding Chimera}" />
+        <local:ItemDisplay Style="{StaticResource SingleCellItemStyle}" Canvas.Left="149" Canvas.Top="307"
```

Comment:
The positions of the splinters are each a little off.  They should be a few pixels higher.  I know it's tough to figure out if you don't own the tab/items, but you can use the bin file I attached in #857 to at least evaluate the positions of the items I own.

I'll try to add some screenshots to demonstrate.  GitHub right now is having a conniption if I try to attach any.

Reply:
I've adjusted them, hopefully it's improved their locations